bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py

```diff
@@ -124,7 +138,6 @@ def set_supplier(source_doc,target_doc,source_parent):
 			"doctype": "Outward Bank Payment",
 			"field_map": {
 				"supplier": "party",
-				"grand_total": "amount",
```

Comment:
Why did you remove this?

Reply:
I thought, for manual reconciliation, the user will enter the total amount as per the allocated amounts in the below payment reference table